src/site/pages/digital/src/containers/MainDesigner/index.tsx

```diff
@@ -108,7 +108,7 @@ export const MainDesigner = ({info, canvas}: Props) => {
                    }}>
             <canvas className="main__canvas"
                     width={w}
-                    height={h-HEADER_HEIGHT} />
+                    height={h-HEADER_HEIGHT+4} />
```

Comment:
Unfortunately, while this *does* fix the issue, it's more important to find out where these 4 pixels come from and make it a constant instead of hard-coding it here. 

I can almost guarantee that now every interaction with the circuit will be 4 pixels off because you also need to update the `camera` size on line 44 of this file, so we do need to be careful with this type of stuff